entity-framework/efcore-and-ef6/index.md

```diff
@@ -1,77 +1,74 @@
+## EF Core
 
-## Entity Framework 6
+Entity Framework Core ([EF Core](../core/index.md)) is a modern object-database mapper for .NET. It supports LINQ queries, change tracking, updates, and schema migrations.
 
-Entity Framework 6 (EF6) is a tried and tested data access technology. It was first released in 2008, as part of .NET Framework 3.5 SP1 and Visual Studio 2008 SP1. Starting with the 4.1 release it has shipped as the [EntityFramework](https://www.nuget.org/packages/EntityFramework/) NuGet package. EF6 runs on the .NET Framework 4.x and .NET Core from 3.0 onwards.
+EF Core works with SQL Server/SQL Azure, SQLite, Azure Cosmos, MySQL, ProstgreSQL, and many more databases through a [database provider plugin model](../core/providers/index.md).
 
-EF6 continues to be a supported product, and will continue to see bug fixes and minor improvements.
+## EF6
 
-## Entity Framework Core
-
-Entity Framework Core (EF Core) is a complete rewrite of EF6 that was first released in 2016. It ships in Nuget packages, the main one being [Microsoft.EntityFrameworkCore](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore/). EF Core is a cross-platform product that runs on .NET Core.
-
-EF Core was designed to provide a developer experience similar to EF6. Most of the top-level APIs remain the same, so EF Core will feel familiar to developers who have used EF6.
+Entity Framework 6 ([EF6](../ef6/index.md)) is an object-relational mapper designed for .NET Framework but with support for .NET Core. EF6 is a stable, supported product, but is no longer being actively developed.
 
 ## Feature comparison
 
-EF Core offers new features that won't be implemented in EF6 (such as [alternate keys](xref:core/modeling/keys#alternate-keys), [batch updates](xref:core/what-is-new/ef-core-1.0#relational-batching-of-statements), and [mixed client/database evaluation in LINQ queries](xref:core/querying/client-eval). But because it's a new code base, it also lacks some features that EF6 has.
+EF Core offers new features that won't be implemented in EF6. However, not all EF6 features are currently implemented in EF Core.
 
-The following tables compare the features available in EF Core and EF6. It's a high-level comparison and doesn't list every feature or explain differences between the same feature in different EF versions.
+The following tables compare the features available in EF Core and EF6. This is a high-level comparison and doesn't list every feature or explain differences between the same feature in different EF versions.
 
 The EF Core column indicates the product version in which the feature first appeared.
 
 ### Creating a model
 
-| **Feature**                                           | **EF 6** | **EF Core**                           |
+| **Feature**                                           | **EF6.4**| **EF Core**                           |
```

Comment:
Is it a good idea to mention the minor version? If we release another version we'd have to update it, and it's not like the EF6 feature set is going to change...

Reply:
@roji I went both ways on this. I didn't want to give the impression that all EF6 releases run on .NET Core, so I started explicitly saying EF6.4 in some places. Then I decided to be consistent.

While it's true that we would need to update this when we do a new release, this is one of the pages we _should_ update with each release anyway.